User.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace common;
+
+use yii\db\ActiveRecord;
+
+class User extends ActiveRecord
+{
+    public static function getCommentsForPosts()
+    {
+        $posts = Post::find()->where(['user_id' => $this->id])->all();
```

Comment:
Have to be taken only IDs not all data.
```suggestion
        $posts = Post::find()->select('id')->where(['user_id' => $this->id])->column();
```